### ISSUE TITLE
Bring Codacy back

### ISSUE THREAD

**Issue by Hans5958** [ScratchAddons/ScratchAddons#2230]
Self-explanatory. Thumbs up if you agree.

Extension of #838, so you should read that if you don't know.

**Comment by apple502j** [comment]
Blocked by #2231

**Comment by Hans5958** [comment]
> 
> 
> Blocked by #2231

Not quite. It does not depend. This can be fixed before or after we get Codacy.

**Comment by Hans5958** [comment]
Just set it up. We're done here.

**Comment by FunctionalMetatable** [comment]
Btw, whats LGTM.com
